### DISCUSSION TITLE
Solid surface temperature probes suddenly drop to the ambient temperature

### DISCUSSION THREAD

**Discussion by DOGzym** [firemodels/fds#12983] [Q&A]
Hi all,

I established PMMA fire spread models under no wind and countercurrent cross wind conditions. However, under windy conditions, the solid surface temperature probe point always drops suddenly to the ambient temperature. It was previously thought that this was due to the complete burning out of the solid below the probe point. However, in the fire spread model without ambient wind, the solid surface temperature output by the probe point will not drop suddenly. This contradicts previous assumptions. So what does the sudden temperature drop in windy conditions mean?

The output of the surface temperature change curves by 7 DEVC-points on the solid surface under windy and windless conditions are given below. 
I‘ve also attached the code as shown below.

[nowind.txt](https://github.com/firemodels/fds/files/15427042/nowind.txt)
[windy.txt](https://github.com/firemodels/fds/files/15427044/windy.txt)
![1](https://github.com/firemodels/fds/assets/126303615/6aec9ff6-5c97-4804-b467-6e5e601d9175)
![2](https://github.com/firemodels/fds/assets/126303615/e111d9cd-ed21-418c-840d-a86ff15a1b46)

**Comment by TristanHehnen** [comment]
For debugging, put also a `DEVC` with `WALL THICKNESS` to the same location of the device recording the `WALL TEMPERATURE` to be sure that the sample is not consumed. The peak shortly before the end let's me assume that the sample is completely consumed.

**Reply by DOGzym** [reply]
Thanks for your suggestion.
In SMOKEVIEW, when the surface temperature of the DEVC suddenly drops, the solid mesh does not completely disappear as expected.
Is there any other reason for this sudden drop in temperature besides complete burnout of the solid mesh?

**Reply by mcgratta** [reply]
Possibly that the THICKNESS of the surface has gone to zero.

**Reply by DOGzym** [reply]
In this case, does it mean that the DEVC point of WALL TEMPERATURE will not move toward the back of the solid as the wall thickness decreases?

**Comment by TristanHehnen** [comment]
I'm having trouble running your case with FDS-6.9.1-0-g889da6a-release. It gets to numerical instabilities within the first few seconds of simulation time.

I did run a case with your material properties and three devices: `WALL TEMPERATURE` (your 'ST1'), `INSIDE WALL TEMPERATURE` (your 'IT1_1') and `WALL THICKNESS`. 

![windy](https://github.com/firemodels/fds/assets/34650649/1617ec9c-8f48-49a3-a4cf-1b1a385d31d3)


My guess is, that you are plotting one of the devices that are recording the `INSIDE WALL TEMPERATURE`. Since they are defined from the front, i.e. with positive `DEPTH` value, they are being pushed into the void when the sample is consumed leading to the default `TMPA` reading. Consider positioning them from the back.

Please check also section "22.2.2 Solid Phase Quantity at a Single Point / Solid Phase Quantity In-Depth" in the [Users Guide](https://github.com/firemodels/fds/releases/download/FDS-6.9.1/FDS_User_Guide.pdf).

**Reply by DOGzym** [reply]
The version I'm currently using is FDS6.7.9_SMV6.7.21_release, I have updated the version to 6.8 and 6.9, but the same code cannot run in these two newer versions.

In your program, when the output THICKNESS becomes 0, will the WALL TEMPERATURE at the same location drop to the default TMPA at the same time? It can indicate whether the solid mesh is completely burned out.

**Reply by TristanHehnen** [reply]
One of the reasons why it does not run in FDS-6.9.1-0-g889da6a-release is the parameter `REFRACTIVE_INDEX` used in the `MATL` definition of PMMA. It seems this parameter is not part of the `MATL` namelist anymore. Still, removing this parameter leads to numerical instabilities for me.

<br><br>
I don't quite follow your question/statement about the thickness and the drop in wall temperature. 
For my plot above, I used your `MATL`, `SURF`, `SPEC`, `REAC` and `DEVC` definitions as provided in `windy.txt`. Since the setup itself gets numerically unstable and takes a long time to run, I've used these definitions in a simplified cone calorimeter setup. I've also added a device tracking the `WALL THICKNESS` in the same cell as `WALL TEMPERATURE` and `INSIDE WALL TEMPERATURE` are recorded, as I suggested you to do earlier. 
I've further adjusted your surface "PMMA CUBE",  by adding a color and an `EXTERNAL_FLUX` to mimic the cone calorimeter setup.

Please do the following:
- run this file  [windy_PMMA.txt](https://github.com/firemodels/fds/files/15454703/windy_PMMA.txt)  with the FDS version you want to use, i.e. "FDS6.7.9_SMV6.7.21_release"
- from the results of this simulation, plot the data from the devices labelled "WT1", "ST1" and "IT1_1" against time
- upload this plot here
